netbout-client/src/main/java/com/netbout/mock/H2Sql.java

```diff
@@ -66,7 +66,7 @@
         final String[] stmts = {
             // @checkstyle LineLength (5 lines)
             "CREATE TABLE alias (name VARCHAR, urn VARCHAR, photo VARCHAR, locale VARCHAR, email VARCHAR)",
-            "CREATE TABLE bout (number BIGINT AUTO_INCREMENT, title VARCHAR, date TIMESTAMP DEFAULT CURRENT_TIMESTAMP)",
+            "CREATE TABLE bout (number BIGINT AUTO_INCREMENT, title VARCHAR, date TIMESTAMP DEFAULT CURRENT_TIMESTAMP, subscription INTEGER)",
```

Comment:
@pinaf u r right it is my mistake. it should be defined on friend table. i fixed it